src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -780,3 +684,496 @@ def refineMesh(cls, meshrefinement, options):
+                                               [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+                        remapEftNodeValueLabel(eft, [3], Node.VALUE_LABEL_D_DS1, [(Node.VALUE_LABEL_D_DS2, [])])
+                    elif e2 == 1:
+                        remapEftNodeValueLabel(eft, [1], Node.VALUE_LABEL_D_DS2,
+                                               [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+                        remapEftNodeValueLabel(eft, [1], Node.VALUE_LABEL_D_DS1, [(Node.VALUE_LABEL_D_DS2, [])])
+
+                elif (e1 == 0) and (e2 == 0):
+                    # Remapping the elements
+                    if e3 == 0:
+                        eft = eftfactory.createEftBasic()
+                        setEftScaleFactorIds(eft, [1], [])
+                        remapEftNodeValueLabel(eft, [4, 8], Node.VALUE_LABEL_D_DS2,
+                                               [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+                        remapEftNodeValueLabel(eft, [4, 8], Node.VALUE_LABEL_D_DS1,
+                                               [(Node.VALUE_LABEL_D_DS2, [])])
+                    elif (e3 == (elementsCount3 - 1)):
+                        nodeIdentifiers[7] = NodeIds[e3 + 1][e2 + 1][e1 + 2]
+                        nodeIdentifiers[5] = NodeIds[e3 + 1][e2][e1 + 2]
+                        eft = eftfactory.createEftBasic()
+                        setEftScaleFactorIds(eft, [1], [])
+                        collapseNodes = [6, 8]
+                        remapEftNodeValueLabel(eft, collapseNodes, Node.VALUE_LABEL_D_DS3,
+                                               [(Node.VALUE_LABEL_D_DS1, []), (Node.VALUE_LABEL_D_DS3, [])])
+                        remapEftNodeValueLabel(eft, [4], Node.VALUE_LABEL_D_DS2,
+                                               [(Node.VALUE_LABEL_D_DS2, []), (Node.VALUE_LABEL_D_DS1, [1])])
+                        remapEftNodeValueLabel(eft, [4], Node.VALUE_LABEL_D_DS1,
+                                               [(Node.VALUE_LABEL_D_DS2, [])])
+
+                elif None in nodeIdentifiers:
+                    continue
+
+                if eft is eftRegular:
+                    element = mesh.createElement(elementIdentifier, elementtemplateRegular)
+                else:
+                    elementtemplateCustom.defineField(coordinates, -1, eft)
+                    element = mesh.createElement(elementIdentifier, elementtemplateCustom)
+                element.setNodesByIdentifier(eft, nodeIdentifiers)
+                if eft.getNumberOfLocalScaleFactors() == 1:
+                    element.setScaleFactors(eft, [-1.0])
+                elementIdentifier += 1
+
+                # Annotation
+                lungMeshGroup.addElement(element)
+                lowerLungMeshGroup.addElement(element)
+                lungSideMeshGroup.addElement(element)
+
+    return elementIdentifier
```

Comment:
Standard is to always have a CR/newline after the last line of code. Please add.